tests/testthat/test-sparsevctrs.R

```diff
@@ -120,3 +120,51 @@ test_that("sparse matrices can be passed to `fit() - xy", {
   # We expect 1 materialization - the outcome
   expect_snapshot(wf_fit <- fit(wf_spec, hotel_data))
 })
+
+test_that("sparse tibble can be passed to `predict()", {
```

Comment:
only doing 1 test since it should be able to predict regardless of how the workflow was created